src/sidebar/Sidebar.tsx

```diff
@@ -14,7 +15,12 @@ export const Sidebar = () => {
   if (noSidebarPathnames.includes(pathname)) return null;
 
   return (
-    <aside className="sidebar">
+    <aside
```

Comment:
Use webcomponents spacing in ContentWrapper as well:
src/components/App/ContentWrapper/ContentWrapper.scss